storybook/.storybook/manager-head.prod.html

```diff
@@ -0,0 +1,28 @@
+<!-- Cookie Consent -->
+<script type="text/javascript" src="https://cdn.cookielaw.org/consent/{{COOKIELAW_ORG}}/OtAutoBlock.js"></script>
+<script type="text/javascript" src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" data-domain-script="{{COOKIELAW_ORG}}"></script>
+<!-- FullStory -->
+<script type="text/plain" class="optanon-category-C0002">
```

Comment:
> The UserGuiding (UG) and FullStory (FS) scripts have the attribute type="text/plain" class="optanon-category-C0002" added to their script tags. If a user consents to performance cookies, these tags are targeted by the cookie consent script, rewritten, and the scripts are executed

Can we document this here? I fear this relationship might be lost down the line

Reply:
I added a comment and a link. It's complicated though. We use cookielaw.org powered by cookierpro brought to you by onetrust.com 🫠